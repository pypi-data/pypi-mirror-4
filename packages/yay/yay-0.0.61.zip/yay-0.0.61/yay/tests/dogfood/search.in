
.search:
  - package://yay/tests/dogfood/search1
  - package://yay/tests/dogfood/search2

.include:
  - somefile.yay
  - onlyin2.yay

