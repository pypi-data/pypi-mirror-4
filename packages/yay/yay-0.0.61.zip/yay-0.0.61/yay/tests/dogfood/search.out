
hello: this one
goodbye: i dont know why

